src/styles/main.scss

```diff
@@ -1 +1,16 @@
-// put code here
+$general-size: 40vmin;
+$hand-width: 3vmin;
+$sec-duration-time: 60s;
+$min-duration-time: 3600s;
```

Comment:
Unused variable:

Both variables $sec-duration-time and $min-duration-time are declared but never used.
Either remove it or give it an use, for example later on in animations.